web-ui/src/App.js

```diff
@@ -52,9 +52,9 @@ function App() {
                 <Header title="Profile" />
                 <Profile />
               </Route>
-              <Route path="/checkins">
-                <Header />
-                <CheckinsPage />
+              <Route path="/personnel">
+                <Header title="My Personnel" />
+                <MyPersonnelPage />
```

Comment:
Instead of making a page for this, can it be integrated into the right hand side of the check-ins page?

It's eventually going to need to look something like:
![Check-in](https://user-images.githubusercontent.com/100510/89488082-adc48a00-d76c-11ea-9840-4182c0f1b4aa.png)

Reply:
Done.
![image](https://user-images.githubusercontent.com/49450715/89535635-4b937580-d7bc-11ea-8266-e6a84aadb86f.png)